modules/vpc-endpoints/variables.tf

```diff
@@ -39,3 +39,43 @@ variable "timeouts" {
   type        = map(string)
   default     = {}
 }
+
+################################################################################
+# Security Group
+################################################################################
+
+variable "create_security_group" {
+  description = "Determines if a security group is created"
+  type        = bool
+  default     = false
```

Comment:
default is `false` so we don't surprise users (like I did with the ALB module 🙃)